### PR TITLE
Mx resources #23, #24, #8 y #7

### PR DESCRIPTION
**Pull request by felipaoo**
se completó el conjunto de llamadas a recursos que aplican para MX #23, #24, #8 y #7

### CODE REVIEW

**Comment by matin** [comment]
@felipaoo there's a conflict in `.gitignore`. Please resolve it.

File: arcus/account.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from arcus.client import Client
```
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/account.py]: I'll will remove it, it was my first model

File: arcus/arcus.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+from arcus.account import Account
```
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/arcus.py]: I'll will remove it, it was my first model

File: arcus/bill.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+from arcus.client import Client
```
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/bill.py]: I'll will remove it, it was my first model

File: arcus/resources/bills.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
+from .base import Resource
+from .transactions import Transaction
+
+
+class Bill(Resource):
+    _endpoint = '/bills'
+
+    def __repr__(self):
```
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/resources/bills.py]: `__repr(self): ` implemented in `Resource ` class.

File: arcus/resources/bills.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        return cls(**bill_dict)
+
+    def pay(self, amount=None):
+        amount = amount or self.balance
+        data = dict(amount=amount, currency='MXN')
+        transaction_dict = self._client.post(f'{self._endpoint}/{self.id}/pay',
+                                             data)
+        return Transaction(**transaction_dict)
+
+    # def get(self, bill_id):
```
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/resources/bills.py]: removed

File: arcus/client.py [hunk 6]
```diff
@@ -53,3 +71,31 @@ def _build_headers(self, endpoint: str, data: dict) -> dict:
...
+        # elif response.status_code == 403:
+        #     raise Forbidden()
+        elif response.status_code == 404:
+            raise NotFound('Resource not found')
+        elif response.status_code == 422:
+            raise UnprocessableEntity('Unprocessable Entity',
+                                      data['code'],
+                                      data['message'],
+                                      data['id'] if 'id' in data else None)
+        # elif response.status_code == 429:
```
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/client.py]: done

File: arcus/account.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/account.py]: removed

File: arcus/arcus.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/arcus.py]: removed

File: arcus/bill.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/bill.py]: removed

File: arcus/resources/transactions.py [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    @classmethod
+    def cancel(cls, transaction_id: int):
+        transaction_data = dict(id=transaction_id)
+        transaction_dict = cls._client.post('/transaction/cancel',
+                                            transaction_data)
+        return Transaction(**transaction_dict)
+
+    @classmethod
+    def get(cls, transaction_id):
+        obj_dict = cls._client\
```
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/resources/transactions.py]: done

File: arcus/transaction.py [hunk 8]
```diff
@@ -0,0 +1,14 @@
+from arcus.client import Client
```
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/transaction.py]: removed

File: arcus/resources/transactions.py (hunk 7 above)
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/resources/transactions.py]: done

File: arcus/client.py [hunk 9]
```diff
@@ -41,10 +48,21 @@ def request(self,
...
-        if response.status_code == 401:
-            raise InvalidAuth('Invalid API authentication credentials')
+        self.check_response(response)
         return response.json()
 
+    @property
+    def account(self):
+        return Account(**self.get('/account'))
+
+    @property
```
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/client.py]: fixed

File: arcus/client.py [hunk 10]
```diff
@@ -41,10 +48,21 @@ def request(self,
...
+    def bills(self):
+        return Bill()
+
+    @property
+    def transactions(self):
+        return Transaction()
```
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/client.py]: changed to `transactions = Transaction`

File: arcus/exc.py [hunk 11]
```diff
@@ -1,5 +1,55 @@
+class ArcusException(Exception):
```
[the remark this answers is not in the document]
Comment (felipaoo) [reply arcus/exc.py]: removed `ArcusException`

**Pull request merged**